services/swarm/config.go

```diff
@@ -0,0 +1,59 @@
+package swarm
+
+import (
+	"github.com/influxdata/kapacitor/services/swarm/client"
+	"github.com/influxdata/kapacitor/tlsconfig"
+	"github.com/pkg/errors"
+)
+
+type Config struct {
+	Enabled bool     `toml:"enabled" override:"enabled"`
+	ID      string   `toml:"id" override:"id"`
+	Servers []string `toml:"servers" override:"servers"`
+	// Path to CA file
+	SSLCA string `toml:"ssl-ca" override:"ssl-ca"`
```

Comment:
Are all of these okay to expose via the API? I see that the other services do this as well, but it feels like there might be some security issue here? Am I wrong?